Practical1/Calculator/Calculator/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Calculator
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int number1;
+            int number2;
+
+            Console.WriteLine("Enter first number: ");
+            number1 = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter second number: ");
+            number2 = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine($"Addition of {number1} and {number2} is {number1 + number2}");
+            Console.WriteLine($"Subtraction of {number1} and {number2} is {number1 - number2}");
+            Console.WriteLine($"Multiplication of {number1} and {number2} is {number1 * number2}");
+            Console.WriteLine($"Division of {number1} and {number2} is {number1 / number2}");
+
+        }
```

Comment:
General Review comments for all 
- bin/obj DLL files should not be checked in 
- Any sort of vsidx files should not be checked in 
- Commit comments should be more descriptive like giving some insights of checkin files/what code is doing.

Suggestion : 
You can take user input which operation user wants instead of giving it all.